### ISSUE TITLE
关于focal loss 中alpha=1是相当于没有对正负样本数量进行加权是吗？

### ISSUE THREAD

**Issue by 123lifei** [Stinky-Tofu/YOLO_V3#79]
@Stinky-Tofu 打扰您 alpha=1, gamma=2 这样的初始化参数设置 是不是只对难以区分的样本进行校正，alpha=1 相当于没有对imbalance加权处理啊 我理解对吗？
我看见论文中是设置的0.25 难道在yolov3中不需要进行正负样本的加权均衡吗  渴望您的指导谢谢

**Comment by Stinky-Tofu** [comment]
@123lifei 我没去调这个参数，所以就用了1

**Comment by chestnutLeeeee** [comment]
这个 focal loss是加在confidence上面的，加在类别上面也没意义。
conf_loss = conf_focal * (
                    alpha+respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox,
logits=conv_raw_conf)
                    (1-alpha)+
                    respond_bgd * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
            )
要加得加在这里，平衡bgd和gt的权重
